client/app/templates/template-explorer.component.js

```diff
@@ -119,11 +127,25 @@ function ComponentController(ListView, TemplatesService, EventNotifications, $st
   }
 
   function getMenuActions() {
-    const menuActions = [];
+    const menuActions = [{
+      name: __('Edit'),
+      title: __('Edit Template'),
+      actionFn: handleEdit,
+    }];
 
     return menuActions;
   }
+  function listActionDisable(config, items) {
+    const menuCreate = 0;
```

Comment:
another nitpick... can 0 and 1 be changed to false and true?  (readability/consistence with in place code base)